docs/configuring-the-deploy-cli.md

```diff
@@ -0,0 +1,135 @@
+# Configuring the Deploy CLI
+
+Configuring the Deploy’s CLI is essential for establishing Auth0 credentials as well as generally modifying the behavior of the tool to your specific needs. There are two ways the Deploy CLI can be configured:
+
+- Configuration file (`config.json`)
+- Environment variables
+
+## Configuration file
+
+A standalone JSON file can be used to configure Deploy CLI. This file will usually reside in the root directory of your project and be called `config.json`.
+
+Example **config.json** file:
+
+```json
+{
+  "AUTH0_DOMAIN": "<YOUR_TENANT_DOMAIN>",
+  "AUTH0_CLIENT_ID": "<YOUR_CLIENT_ID>",
+  "AUTH0_ALLOW_DELETE": false
+}
+```
+
+> ⚠️ **NOTE:** Hard-coding credentials is not recommended, and risks secret leakage should this file ever be committed to a public version control system. Instead, passing credentials as [environment variables](#) is considered best practice.
+
+### Environment variables
```

Comment:
If there are two different ways to configure the tool, and the first one is an H2 (`##`), should the second one be an H2 as well?

Reply:
Yeah, this is a good point. Thanks!